src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -259,6 +259,20 @@ Apply to thead to make headers not bold.
     padding-top: 0;
 }
 
+.question-recommended-length {
```

Comment:
I was trying to follow the alphabetical order. And it seems there are not such classes defined in this  file. Otherwise I can shift these classes to the starting or end if that makes sense ?